notebooks/combined_wind_wave_surge_building_dmg.ipynb

```diff
@@ -2,9 +2,13 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
I see weird execution count everywhere... haha how did you remove the output cell???
<img width="195" alt="image" src="https://user-images.githubusercontent.com/13950475/221040429-b0325493-5f2b-4c45-8651-228ae3b05dd6.png">


Could you try Kernel "restart and clear output"?
<img width="258" alt="image" src="https://user-images.githubusercontent.com/13950475/221040330-380bdd56-a6cb-4377-8508-0fd370338791.png">